crates/store/re_grpc_client/src/redap/address.rs

```diff
@@ -64,6 +68,40 @@ impl Scheme {
             Self::RerunHttp => "http",
         }
     }
+
+    /// Converts a rerun url into a canonical http or https url.
+    fn canonical_url(&self, url: &str) -> String {
+        match self {
+            Self::Rerun => {
+                debug_assert!(url.starts_with("rerun://"));
+                url.replace("rerun://", "https://")
+            }
+            Self::RerunHttp => {
+                debug_assert!(url.starts_with("rerun+http://"));
+                url.replace("rerun+http://", "http://")
+            }
+            Self::RerunHttps => {
+                debug_assert!(url.starts_with("rerun+https://"));
+                url.replace("rerun+https://", "https://")
+            }
+        }
+    }
+}
+
+impl TryFrom<&str> for Scheme {
+    type Error = InvalidScheme;
+
+    fn try_from(url: &str) -> Result<Self, Self::Error> {
+        if url.starts_with("rerun://") {
+            Ok(Self::Rerun)
```

Comment:
Do we need to distinguish `rerun://` from `rerun+https://`, or can we just return `RerunHttps` here and remove the `Rerun` alternative?

Reply:
If we want to restore the original URL (for example for debugging) we need to have all variants.